templates/profile/prometheus/rules.yml.erb

```diff
@@ -160,6 +160,33 @@ groups:
     annotations:
       summary: 'Dark Blue repository storage, {{$labels.device}} is out of available space.'
       description: '{{$labels.device}} has had less than 100GB of available space for more than 5 minutes.'
+  - alert: MysqlDown
+    expr: >
+      mysql_up == 0
+    for: 5m
+    labels:
+      severity: page
+    annotations:
+      summary: 'MySQL is DOWN on {{$lables.hostname}}'
+      description: 'MySQL is DOWN on {{$labels.hostname}}'
+  - alert: MysqlSlaveSqlDown
+    expr: >
+      mysql_slave_status_slave_sql_running  == 0
+    for: 5m
+    lablels:
+      severity: page
+    annotations:
+      summary: 'MySQL Slave SQL is Not Running'
+      description: 'MySQL Slave SQL is not Running on {{$labels.hostname}}'
```

Comment:
Second 'SQL' is redundant

Reply:
For what it's worth, we don't use the description at all. We only use the summary, so if the slave isn't running on two different servers, we'll receive an alert that looks like this:

```
MySQL Slave SQL is Not Running
MySQL Slave SQL is Not Running
```

I think the hostname should be in the summary, and I don't really have any feedback around what should or shouldn't go into a description, because we don't use it.

Reply:
Fixed and learned that summary is used, description is not.